src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj

```diff
@@ -112,8 +112,8 @@
            Similarly, we set CopyToOutputDirectory for the regular build and PackageCopyToPutput for the packaging process. -->
       <Content Include="%(NetFrameworkBuildHostAssets.Identity)"
                Condition="'%(NetFrameworkBuildHostAssets.TargetPath)' != '' and '%(NetFrameworkBuildHostAssets.Extension)' != '.xml'"
-               TargetPath="%(NetFrameworkBuildHostAssets.ContentFolderName)\%(NetFrameworkBuildHostAssets.TargetPath)"
-               PackagePath="contentFiles\any\any\%(NetFrameworkBuildHostAssets.ContentFolderName)\%(NetFrameworkBuildHostAssets.TargetPath)"
+               TargetPath="$([System.IO.Path]::Combine('%(NetFrameworkBuildHostAssets.ContentFolderName)', '%(NetFrameworkBuildHostAssets.TargetPath)'))"
+               PackagePath="$([System.IO.Path]::Combine('contentFiles', 'any', 'any', '%(NetFrameworkBuildHostAssets.ContentFolderName)', '%(NetFrameworkBuildHostAssets.TargetPath)'))"
```

Comment:
@baronfel type of stuff an msbuild analyzer / warning would help with.

Reply:
Seriously.

Reply:
Interesting - I thought we were more or less slash agnostic in most of MSBuild. Unless you're wanting a stricter rule of 'no direct path creation allowed'?

Reply:
This isn't a globbing pattern, it's a relative path. It ends up as metadata and eventually fed into the R2R tasks, which path.combines it with another parameter. On Linux, at that point the task the Path.Combine sees the backslash and does what it should on Linux (not treat it as a directory separator char).

If it was a globbing pattern, then MSBuild would fix it up.